### ISSUE TITLE
Investigate bug where saving does not work when deleting a drawn component line

### ISSUE THREAD

**Issue by mddilley** [cityofaustin/atd-data-tech#18590]
We received a report where a component line delete failed to save when editing an existing component. This happened on project 1989, and I included the steps to replicate from the report below.

> I ran into a weird thing this morning that has never been an issue. I tried editing mapping and it wouldn't let me save. 
> The edit was trying to save was deleting a hand drawn line and selecting one of the street network lines. The save button was unresponsive. 

> I am able to edit tabular data. I was able to edit a point. I was able to select a different link and safe. So it seems like there is something wrong with this link of the trail over the UP tracks.
> I was able to just adjust the custom mapping and avoid using this segment. Not a high priority issue upon further exploration just a strange dead / unselectable link in Moped

**Screenshot submitted**
![image](https://github.com/user-attachments/assets/3fc9f545-37d7-411d-a779-dd97d27b7509)

**Comment by mddilley** [comment]
Looping back here to note that this issue is happening because the CTN street segments feature service is returning a MultiLinestring geometry instead of a Linestring. The GraphQL error is happening because the app doesn't handle this unexpected type.

I shared this with Andrew, David, Frank, and Charlie yesterday and learned some details:
- CTN features will be updated soon (Andrew)
   - so, maybe this will resolve after the update
- Charlie pointed out that this is an urban trail (north of Stoneway Drive and Great Northern Boulevard)
- There were some good theories from Frank and David that this could have occurred when an intersecting line "split" into a MultiLinestring

**Comment by mddilley** [comment]
Setting this as blocked by another issue concerning surfacing map errors. This is another one where we don't show any user feedback when an action fails unexpectedly. We should use the same error pattern here.
